src/Microsoft.IdentityModel.JsonWebTokens/Microsoft.IdentityModel.JsonWebTokens.csproj

```diff
@@ -29,12 +29,12 @@
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net472' Or '$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'net6.0'">
     <PackageReference Include="System.Text.Encoding" Version="$(SystemTextEncoding)" />
-	<PackageReference Include="System.Text.Encodings.Web" Version="$(SystemTextEncodingsWeb)" />
-    <PackageReference Include="System.Text.Json" Version="$(SystemTextJson)" />
+
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net45' Or '$(TargetFramework)' == 'net461' Or '$(TargetFramework)' == 'net462' Or '$(TargetFramework)' == 'net472'">
```

Comment:
Here and elsewhere, would 

```
<ItemGroup Condition="'$(TargetFrameworkIdentifier)' != 'NETCoreApp'">
```
be a more succinct way of expressing this? I'm not 100% clear on how that would interact with netstandard2.0.

I was going to update this but worried there may be some implications I didn't understand.